### DISCUSSION TITLE
Where do docs go?

### DISCUSSION THREAD

**Discussion by akvadrako** [Igalia/cog#291] [Q&A] [answered]
So I found by looking at the source there are some settings:

- `COG_PLATFORM_FDO_VIEW_WIDTH`
- `COG_PLATFORM_FDO_VIEW_HEIGHT`

I see in the README you are asking for help with docs and I thought it would be nice to list these somewhere. My question is where should the docs go?

Is it the github wiki, the docs/ directory, source code comments or something like `USAGE.md` in the root?

**Comment by aperezdc** [comment, marked as answer]
Hello @akvadrako! There are two kinds of documentation:

- Reference documentation is in comments inside the code itself, look for comments starting with `/**`.
- Other documentation will go inside the `docs/` subdirectory, with a Markdown file for each topic. 

Currently in the `master` branch there is nothing under `docs/`, but you can see in PR #288 some content is being added. Once that pull request is merged (hopefully soon!), you could modify the `docs/platform-fdo.md` file to contribute some documentation about the FDO platform plug-in—including a list of environment variables and a description of what they do.+

After new documentation is merged into the `master` branch, it will take only a few minutes to appear published at https://igalia.github.io/cog/ 😃
